### PR TITLE
Ping/create edit reminder

### PR DESCRIPTION
**Pull request by Awesomeman007**
- working create reminder screen
- working edit reminder screen

### CODE REVIEW

**Comment by pointmekin** [comment]
Consider adding the new files from .idea and android folders to .gitignore as some of our android directory are slightly different and im afraid some changes can cause the app to fail to build on someone’s device.

File: src/components/organisms/ReminderForm.tsx [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+  };
+
+  const onDateTimeChange = (event, selectedDate?: Date | undefined) => {
+    if (!selectedDate) return;
+    const currentDate = selectedDate.getTime();
+    setShowDateTimePicker(false);
+    setDate && setDate(new Date(currentDate));
+  };
+
+  // const onTimeChange = (event, selectedTime?: Date | undefined) => {
```
Comment (pointmekin) [inline src/components/organisms/ReminderForm.tsx]: consider removing commented code

File: src/internationalization/translations/th.json [hunk 2]
```diff
@@ -458,5 +458,38 @@
     "3month": "3 เดือน",
     "year": "1 ปี",
     "allTime": "ทั้งหมด"
+  },
+  "reminder": {
+    "createReminder": "สร้างการเตือน",
+    "editReminder": "แก้ไชการเตือน",
```
Comment (pointmekin) [inline src/internationalization/translations/th.json]: typo

File: src/internationalization/translations/th.json [hunk 3]
```diff
@@ -458,5 +458,38 @@
...
+    "create": "สร้าง",
+    "edit": "แก้ไช",
```
Comment (pointmekin) [inline src/internationalization/translations/th.json]: typo

File: src/screens/reminder/CreateReminderScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+  const [date, setDate] = useState<Date>(new Date());
+  const [image, setImage] = useState<ImagePickerResponse>();
+  const [notifyMyCaretakers, setNotifyMyCaretaker] = useState<boolean>(true);
+  const [repeatition, setRepeatition] = useState<string>('doesNotRepeat');
+
+  const handleButtonState = () => {
+    if (watchInputs.title || watchInputs.title === '') return false;
+    return false;
+  };
```
Comment (pointmekin) [inline src/screens/reminder/CreateReminderScreen.tsx:30]: wont this always return false? or is it temporary?

**Review by bweam — APPROVED** [review]

**Pull request merged**